hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,8 @@
+RELEASE_TYPE: patch
+
+This release removes some functionality from the shrinker that was taking a
```

Comment:
Is this the correct release note?  Because I'm not seeing removed functionality...

Reply:
Ugh it is not. Thanks.

I'm really feeling the limitations of our `RELEASE.rst` system for many stacked pull requests right now.